lava/lava-job-definitions/shared/templates/bootbomb-recovery.yaml

```diff
@@ -47,20 +47,14 @@
            - cd /lava-lxc
            - dpkg -i imx-usb-loader_0-git20181105.4aa98090-1_amd64.deb
            - lsusb
+           - grep 15a2 /sys/bus/usb/devices/*/idVendor
```

Comment:
What's 15a2?

Reply:
That is the vendor ID if the interface, which is raised when the device is powered up in recovery mode. This ID is used by the bootbomb (imx_usb) https://lava.mbedcloudtesting.com/scheduler/job/71738#L900. We use it to make sure the device is started up in recovery mode and then use the LAVA_STORAGE_INFO_0_SATA to filter out the correct USB path https://github.com/ARMmbed/mbl-tools/pull/342/files#diff-566d7644a152dea8a58340bf405d528bR51

Reply:
And this line is actually just for debug from by healthcheck job

Reply:
I guess we can remove the grep and add a comment with that you've just said.

Reply:
Removed the grep and added a comment.